packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -557,70 +561,28 @@ void main() {
       ),
     ));
 
-    final Offset floatingActionButtonOriginBottomCenter = tester.getCenter(find.byType(FloatingActionButton));
+    // Get the Rect of the FAB to compare after the SnackBar appears.
+    final Rect originalFabRect = tester.getRect(find.byType(FloatingActionButton));
 
     await tester.tap(find.text('X'));
     await tester.pump(); // start animation
     await tester.pump(const Duration(milliseconds: 750)); // Animation last frame.
 
-    final Offset snackBarTopCenter = tester.getCenter(find.byType(SnackBar));
-    final Offset floatingActionButtonBottomCenter = tester.getCenter(find.byType(FloatingActionButton));
+    final Rect fabRect = tester.getRect(find.byType(FloatingActionButton));
 
-    expect(floatingActionButtonOriginBottomCenter.dy > floatingActionButtonBottomCenter.dy, true);
-    expect(snackBarTopCenter.dy > floatingActionButtonBottomCenter.dy, true);
-  });
+    // FAB should shift upwards after SnackBar appears.
+    expect(originalFabRect.center.dy > fabRect.center.dy, true);
```

Comment:
Use [lessThan](https://master-api.flutter.dev/flutter/package-matcher_matcher/lessThan.html) matcher here:
```suggestion
    expect(fabRect.center.dy, lessThan(originalFabRect.center.dy));
```

Reply:
Fixed